packages/ui/src/components/Table/Table.stories.tsx

```diff
@@ -367,3 +371,510 @@ export const CellProfile: Story = {
     );
   },
 };
+
+export const SelectionSingleToggle: Story = {
+  render: () => {
+    const [selectedKeys, setSelectedKeys] = useState<Selection>(new Set([]));
+
+    return (
+      <Table
+        selectionMode="single"
+        selectionBehavior="toggle"
+        selectedKeys={selectedKeys}
+        onSelectionChange={setSelectedKeys}
+      >
+        <TableHeader>
+          <Column isRowHeader>Name</Column>
+          <Column>Owner</Column>
+          <Column>Type</Column>
+        </TableHeader>
+        <TableBody>
+          <Row id="1">
+            <Cell title="Component Library" />
+            <Cell title="Design System" />
+            <Cell title="library" />
+          </Row>
+          <Row id="2">
+            <Cell title="API Gateway" />
+            <Cell title="Platform" />
+            <Cell title="service" />
+          </Row>
+          <Row id="3">
+            <Cell title="Documentation Site" />
+            <Cell title="DevEx" />
+            <Cell title="website" />
+          </Row>
+        </TableBody>
+      </Table>
+    );
+  },
+};
+
+export const SelectionMultiToggle: Story = {
+  render: () => {
+    const [selectedKeys, setSelectedKeys] = useState<Selection>(new Set([]));
+
+    return (
+      <ResizableTableContainer>
```

Comment:
Why do we need `ResizableTableContainer` ? Is this just for the example or part of the recommendation?